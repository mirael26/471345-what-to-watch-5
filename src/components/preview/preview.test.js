import React from "react";
import renderer from "react-test-renderer";

import Preview from "./preview";

it(`Should Preview render correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <Preview
          onVideoMouseOver={noop}
          onVideoMouseOut={noop}
        />
        , {
          createNodeMock: () => {
            return {};
          }
        }
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
