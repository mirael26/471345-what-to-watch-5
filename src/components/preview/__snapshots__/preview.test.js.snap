// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Preview render correctly 1`] = `
<div
  className="small-movie-card__image"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
/>
`;
